packages/app/src/webapp/views/sidecar.ts

```diff
@@ -403,7 +403,7 @@ export const setMaximization = (tab: Tab, op = 'add', cause: MaximizationCause =
   }
 
   tab.classList[op]('sidecar-full-screen')
-  setTimeout(() => eventBus.emit('/sidecar/maximize'), 850)
+  eventBus.emit('/sidecar/maximize')
```

Comment:
we need the setTimeout, but the timeout value can probably be zero

Reply:
updated, thanks!